### PR TITLE
Playbook migration for remcos

### PR DESCRIPTION
**Pull request by ljstella**
In preparation for upcoming content release, migrating 3 playbooks to `security_content`

### CODE REVIEW

File: playbooks/block_indicators.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+references: []
+app_list:
+- "Palo Alto Networks Firewall"
+- "CarbonBlack Response"
+- "OpenDNS Umbrella"
+tags:
+  platform_tags:
+  - Response
+  playbook_fields:
+  - DestinationDnsDomain
```
Comment (philroyer-phantom) [inline playbooks/block_indicators.yml]: I think these three fields should use the same capitalization as SOAR uses for all CEF fields, so they should be destinationDnsDomain, destinationAddress, and fileHash

File: playbooks/email_notification_for_malware.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+app_list:
+- "VirusTotal"
+- "WildFire"
+- "CarbonBlack Response"
+- "SMTP"
+tags:
+  platform_tags:
+  - Response
+  playbook_fields:
+  - FileHash
```
Comment (philroyer-phantom) [inline playbooks/email_notification_for_malware.yml]: Should be fileHash because SOAR is case-sensitive

Comment (philroyer-phantom) [reply playbooks/email_notification_for_malware.yml]: Should also have the vaultId field which is used for "detonate file"

File: playbooks/malware_hunt_and_contain.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+app_list:
+- "LDAP"
+- "ServiceNow"
+- "CarbonBlack Response"
+- "VirusTotal"
+tags:
+  platform_tags:
+  - Response
+  playbook_fields:
+  - FileHash
```
Comment (philroyer-phantom) [inline playbooks/malware_hunt_and_contain.yml]: Should be filehash

**Review by philroyer-phantom — CHANGES_REQUESTED** [review]
Just some small changes around playbook_fields then it should be good

File: playbooks/block_indicators.yml (hunk 1 above)
Comment (ljstella) [reply playbooks/block_indicators.yml]: Fixed

File: playbooks/email_notification_for_malware.yml (hunk 2 above)
Comment (ljstella) [reply playbooks/email_notification_for_malware.yml]: Fixed

File: playbooks/malware_hunt_and_contain.yml (hunk 3 above)
Comment (ljstella) [reply playbooks/malware_hunt_and_contain.yml]: Fixed

**Review by philroyer-phantom — APPROVED** [review]
👍

**Review by josehelps — APPROVED** [review]
LGTM

**Pull request merged**